src/styles/index.css

```diff
@@ -0,0 +1,61 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: grey;
```

Comment:
The background color 'grey' is a named color and might not be consistent across different browsers. Consider using a hexadecimal or RGB color value for consistency.